src/app/(pages)/pool/new/create-pool-form.tsx

```diff
@@ -148,7 +148,7 @@ export default function CreatePoolForm() {
                     setFormErrors({})
                     setIsSubmitting(false)
                 }}
-                className='flex w-full flex-col gap-6 py-6'>
+                className='flex h-full w-full flex-col gap-6 overflow-y-auto pb-[90px] pt-6'>
```

Comment:
we can use size-full as shortcut to h-full w-full, will be applied automatically next time we touch this file

Reply:
corrected